### DISCUSSION TITLE
v3.1.0

### DISCUSSION THREAD

**Discussion by klaasnicolaas** [klaasnicolaas/home-assistant-glow#169] [Announcements]
## ✨ New features

- #144 Add a Version Sensor @klaasnicolaas

## 🚀 Enhancements

- #130 Rename the case files @klaasnicolaas
- #141 Migrate to new board platform configuration @pschneider87
- #145 Change the used time platform @klaasnicolaas

## 🧰 Maintenance

- #108 Fix broken firmware publish @klaasnicolaas
- #119 🎆Updates maintenance/license year to 2022 @klaasnicolaas
- #124 Stale workflow to cleanup issues @klaasnicolaas

## 📚 Documentation

- #118 📚 Update documentation @klaasnicolaas
- #145 Change the used time platform @klaasnicolaas
- #168 📚 Update on the documentation @klaasnicolaas

## ⬆️ Dependency updates

<details>
<summary>10 changes</summary>

- #117 Bump release-drafter/release-drafter from 5.15.0 to 5.16.1 @dependabot
- #116 Bump esphome/build-action from 1.1 to 1.2.1 @dependabot
- #123 Bump release-drafter/release-drafter from 5.16.1 to 5.17.5 @dependabot
- #125 Bump release-drafter/release-drafter from 5.17.5 to 5.17.6 @dependabot
- #129 Bump release-drafter/release-drafter from 5.17.6 to 5.18.0 @dependabot
- #131 Bump release-drafter/release-drafter from 5.18.0 to 5.18.1 @dependabot
- #138 Bump actions/checkout from 2 to 3 @dependabot
- #143 Bump release-drafter/release-drafter from 5.18.1 to 5.19.0 @dependabot
- #149 Bump esphome/build-action from 1.2.1 to 1.3.1 @dependabot
- #161 Bump actions/stale from 4 to 5 @dependabot
</details>

**Full Changelog**: https://github.com/klaasnicolaas/home-assistant-glow/compare/v3.0.0...v3.1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/klaasnicolaas/home-assistant-glow/releases/tag/v3.1.0'>v3.1.0</a>.</em>

**Comment by Eddie46848** [comment]
Hi
The version number is still 3.0.0 in the yaml, so new sensor still shows version 3.0.0.

**Reply by klaasnicolaas** [reply]
Oh dear how stupid of me that I didn't adjust that 😅
